toplevel/coqtop.ml

```diff
@@ -139,12 +139,16 @@ let ltac_debug_answer = let open DebugHook.Answer in function
       Format.fprintf !Topfmt.err_ft "@[%a@]@\n%!" Pp.pp_with (str "Goal:" ++ fnl () ++ g)
     | Output o ->
       Format.fprintf !Topfmt.err_ft "@[%a@]@\n%!" Pp.pp_with o
+    | Init ->
+      Format.fprintf !Topfmt.err_ft "@[%a@]@\n%!" Pp.pp_with (str "Init")
+    | Stack _ -> failwith "Stack" (* not supported in coqtop *)
+    | Vars _ -> failwith "Vars" (* not supported in coqtop *)
```

Comment:
Why is this not supported? You could print it to the terminal? I'd suggest using a proper error message, not an anomaly.

Reply:
There's currently no code path that can generate these replies in coqtop.  The command line debugger is a candidate for removal--its usability is so much less than the visual debugger that it's probably not worth keeping around at all, let alone enhancing it.

Reply:
you should assert false or anomaly for things that are never supposed to execute, not failwith

Reply:
What do you have against `failwith`?

I changed it to `anomaly`.  I don't like `assert false` because it doesn't return a message string (unlike `failwith`).  At best you get a file and a line number in a stack trace, which you then need to look up in the exactly matching source code to learn anything, so harder for both developers and users.